world_universities_and_domains.json

```diff
@@ -44915,13 +44915,13 @@
   },
   {
     "web_pages": [
-      "http://www.accrapolytechnic.edu.gh/"
+      "https://atu.edu.gh/"
     ],
-    "name": "Accra Polytechnic",
+    "name": "Accra Technical University",
     "alpha_two_code": "GH",
     "state-province": null,
     "domains": [
-      "accrapolytechnic.edu.gh"
+      "https://atu.edu.gh/"
```

Comment:
Domains should not include the https:// part. Also the trailing slash should be removed from the domain entries.